docs/data-sources/eps_enterprise_projects.md

```diff
@@ -0,0 +1,60 @@
+---
+subcategory: "Enterprise Project Management Service (EPS)"
+---
+
+# huaweicloud_enterprise_projects
+
+Use this data source to get the list of EPS enterprise projects.
+
+## Example Usage
+
+```hcl
+variable "name" {}
+
+data "huaweicloud_enterprise_projects" "test" {
+  name = var.name
+}
+```
+
+## Argument Reference
+
+The following arguments are supported:
+
+* `name` - (Optional, String) Specifies the enterprise project name. Fuzzy search is supported.
+
+* `enterprise_project_id` - (Optional, String) Specifies the ID of an enterprise project.
+  The value **0** indicates enterprise project default.
+
+* `status` - (Optional, Int) Specifies the status of an enterprise project. The valid values are as follows:
+  + **1** indicates Enabled.
+  + **2** indicates Disabled.
+
+* `type` - (Optional, String) Specifies the type of an enterprise project. The valid values are as follows:
+  + **prod** indicates Commercial project.
+  + **poc** indicates Test project.
+
+## Attribute Reference
+
+In addition to all arguments above, the following attributes are exported:
+
+* `id` - Data source ID.
+
+* `enterprise_projects` - All groups that match the filter parameters.
+  The [enterprise_projects](#Enterprise_Projects) structure is documented below.
+
+<a name="Enterprise_Projects"></a>
+The `groups` block supports:
```

Comment:
The `enterprise_projects` block supports:

Reply:
fixed